crates/turborepo-ui/src/tui/app.rs

```diff
@@ -77,6 +77,10 @@ impl<I> App<I> {
     pub fn term_size(&self) -> (u16, u16) {
         self.pane.term_size()
     }
+
+    pub fn update_tasks(&mut self, tasks: Vec<String>) {
+        self.table = TaskTable::new(tasks.clone());
```

Comment:
Due to a prior hack, there should be `self.next()` as otherwise the table might be out of sync with the pane. i.e. display wrong pane for selected task in table